ui/snippets/header/Header.tsx

```diff
@@ -42,7 +45,12 @@ const Header = ({ isHomePage, renderSearchBar }: Props) => {
         >
           <Burger/>
           <NetworkLogo/>
-          { config.features.account.isEnabled ? <ProfileMenuMobile/> : <Box boxSize={ 10 }/> }
+          <Box display="flex">
+            { config.features.account.isEnabled ? <ProfileMenuMobile/> : <Box boxSize={ 10 }/> }
+            <Box ml={ 2 }>
+              <WalletMenuMobile/>
+            </Box>
+          </Box>
```

Comment:
```suggestion
          <Flex columnGap={ 2 }>
            { config.features.account.isEnabled ? <ProfileMenuMobile/> : <Box boxSize={ 10 }/> }
            <WalletMenuMobile/>
          </Flex>
```

Reply:
we can simplify here a little

Reply:
Don't we need to check here what `blockchainInteraction` feature is enabled, do we? I guess we should not display wallet menu if WallectConnect is not configured for an instance